utils/box-symbol.stanza

```diff
@@ -244,6 +244,22 @@ public defn make-box-symbol (banks:Seqable<BoxBank>) :
 ;=======================================================<doc>
 
 public defn make-box-symbol () :
+  ;Sort the given pins by their .pin-properties-row-index field.
```

Comment:
All the logic here in `box-symbol.stanza` can be removed. It has been superceded by another implementation in a different file.